Duplicati/Library/Main/Operation/RepairHandler.cs

```diff
@@ -304,137 +304,141 @@ public void RunRepairRemote()
                             }
                             else if (n.Type == RemoteVolumeType.Index)
                             {
-                                var w = new IndexVolumeWriter(m_options);

```

Comment:
The following hunks were due to having MonoDevelop enclose the variables in a `using` block.  Unfortunately, this resulted in a messy diff.